index.js

```diff
@@ -115,6 +117,7 @@ const ContainerService = {
   _identitiesState: {},
   _windowMap: new Map(),
   _containerWasEnabled: false,
+  _onThemeChangedCallbacks: [],
```

Comment:
Does this need to be an array of callbacks? I only see a single callback that we're adding here. I.e., `port.postMessage` to send the `theme, topic` to the background `themeManager`?